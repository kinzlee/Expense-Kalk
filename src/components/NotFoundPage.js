import React from 'react';
import {Link} from 'react-router-dom';


const NotFoundPage = () => (
    <div>
        page not found 404! <Link to="/" >go back to homePage</Link>
    </div>
);

export default NotFoundPage;
