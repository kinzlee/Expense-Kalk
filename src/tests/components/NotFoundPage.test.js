import React from 'react';
import {shallow} from 'enzyme';
import ExpensesListItem from '../../components/NotFoundPage';
import NotFoundPage from '../../components/NotFoundPage';
// import expenses from '../fixture/expenses';

test('should render not found page correctly', () => {
    const wrapper = shallow(<NotFoundPage/>);
    expect(wrapper).toMatchSnapshot();  
});