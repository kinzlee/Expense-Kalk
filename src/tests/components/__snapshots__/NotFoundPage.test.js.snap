// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render not found page correctly 1`] = `
<div>
  page not found 404! 
  <Link
    to="/"
  >
    go back to homePage
  </Link>
</div>
`;
